src/index.js

```diff
@@ -649,28 +668,49 @@ async function processImageForScale(imageBuffer, scale, format, quality, maxScal
 /**
  * Create Contents.json for iOS assets
  */
-function createContentsJson(name, type = 'image') {
+function createContentsJson(name, format = 'png') {
+  if (format === 'svg') {
+    return JSON.stringify({
+      images: [
+        {
+          filename: `${name}.${format}`,
+          idiom: 'universal'
+        }
+      ],
+      info: {
+        author: 'Figma Asset Downloader',
+        version: 1
+      },
+      properties: {
+        "template-rendering-intent": "original"
+      }
+    }, null, 2);
+  }
+
   return JSON.stringify({
     images: [
       {
-        filename: `${name}.png`,
+        filename: `${name}.${format}`,
         idiom: 'universal',
         scale: '1x'
       },
       {
-        filename: `${name}@2x.png`,
+        filename: `${name}@2x.${format}`,
         idiom: 'universal',
         scale: '2x'
       },
       {
-        filename: `${name}@3x.png`,
+        filename: `${name}@3x.${format}`,
         idiom: 'universal',
         scale: '3x'
       }
     ],
     info: {
-      author: 'Figma Asset Downloader',
-      version: 1
+      version: 1,
```

Comment:
Right now it’s missing handling for the ~ipad case.

Reply:
I thought universal covers either device?

Reply:
This is an example

```
{
  "info" : {
    "author" : "xcode",
    "version" : 1
  },
  "images" : [
    {
      "filename" : "button_context_curve~ipadL@1x.png",
      "idiom" : "ipad",
      "scale" : "1x"
    },
    {
      "scale" : "2x",
      "filename" : "button_context_curve~ipadL@2x.png",
      "idiom" : "ipad"
    },
    {
      "scale" : "1x",
      "filename" : "button_context_curveL@1x.png",
      "idiom" : "universal"
    },
    {
      "filename" : "button_context_curveL@2x.png",
      "scale" : "2x",
      "idiom" : "universal"
    },
    {
      "scale" : "3x",
      "idiom" : "universal",
      "filename" : "button_context_curveL@3x.png"
    }
  ]
}
```

Reply:
Opened #5 to fix this